atlp-rwanda/matadors-rn-medica
# Feature: add onboarding screens

#### What does this PR do?
This pull request adds updates to the onboarding screens of the medical app to ensure compatibility with both dark and light modes. It includes adjustments to the UI elements, colors, and layouts to maintain consistency and readability across different modes.

#### Description of Task to be completed?

The task involved reviewing the existing onboarding screens and making necessary adjustments to ensure they are visually appealing and functional in both dark and light modes. This includes modifying color schemes, updating text styles, and adjusting layout spacing where necessary.

#### How should this be manually tested?

These changes could be tested by running the app on your Android Studio emulator or by having the Expo app installed on your phone and scanning the QR code to try the app.

1. Open the app in light mode and go through the onboarding process.
2. Verify that all text and UI elements are visible and readable.
3. Switch to dark mode and repeat the onboarding process.
4. Ensure that the onboarding screens maintain their usability and readability in dark mode as well.
5. Test on different devices with varying screen sizes and resolutions to ensure consistency.

#### Any background context you want to provide?
The update to the onboarding screens was necessary to enhance the user experience and ensure accessibility for users who prefer dark mode. By making these adjustments, we aim to provide a seamless onboarding experience for all users, regardless of their preferred mode.


#### Screenshots

<div style="display: flex; flex-wrap: wrap;">
    <img src="https://github.com/atlp-rwanda/matadors-rn-medica/assets/105460631/774b8eef-a805-454f-9424-599787eec6ce" alt="WhatsApp Image 1" height="480px">
    <img src="https://github.com/atlp-rwanda/matadors-rn-medica/assets/105460631/254bafed-4233-4871-9e38-12cd0238f502" alt="WhatsApp Image 2" height="480px">
</div>
